src/app.js

```diff
@@ -183,6 +185,24 @@ async function backgroundWorker () {
         }
       }
     }
+    // After OTP resets, the next thing to check for is are there any other notifications.
+    const notification = await checkForNotifications()
+    if (notification) {
+      const dateOne = new Date(Date.now() + 1000)
```

Comment:
One second in the future? Seems good, but maybe a comment would be good here - I saw the `+ 1000`, and at first I thought it was the `* 1000` we are constantly doing to go from seconds to ms.